splice/models.py

```diff
@@ -95,16 +97,18 @@ class Adgroup(db.Model):
     id = db.Column(db.Integer(), autoincrement=True, primary_key=True, info={"identity": [1, 1]})
     locale = db.Column(db.String(14), nullable=False)
 
-    type = db.Column(db.String(16))
-    paused = db.Column('paused', db.Boolean(), nullable=False, server_default=db.text(u'false'))
```

Comment:
I agree. I thinks it's also necessary to name every single adgroup, even for those DT ones.

We can fill up the existing data with the default value in the migration script.